OpenRA.Mods.Common/Traits/Player/PlayerStatistics.cs

```diff
@@ -84,7 +86,9 @@ void UpdateArmyThisMinute()
 
 		void ITick.Tick(Actor self)
 		{
-			if (self.World.WorldTick % 1500 == 1)
+			var timestep = self.World.IsReplay ? replayTimestep : self.World.Timestep;
+
+			if (timestep * self.World.WorldTick % 60000 == 0)
```

Comment:
This breaks if 60000 can not be divided properly with the timestep.  Example case: fast gamespeed in RA.